appcues/src/main/java/com/appcues/data/model/ExperiencePrimitive.kt

```diff
@@ -31,11 +31,17 @@ internal sealed class ExperiencePrimitive(
         override val id: UUID,
         override val style: ComponentStyle = ComponentStyle(),
         val text: String,
+        val spans: List<TextSpanPrimitive>,
```

Comment:
can we get rid of `text` entirely at this point?

Reply:
it is possible, but its still useful to have either "text" the way it is, or replace  it entirely and use the contentDescription when we need the full text

Reply:
is there anywhere that we still use this `text` value after mapping to spans?